deps.sh

```diff
@@ -30,7 +30,7 @@ BUILDDIR=${BUILDDIR:-${SELFDIR}/deps/build}
 echo "Building in ${BUILDDIR}"
 echo "Installing to ${DESTDIR}"
 
-if [ ! -e ${DESTDIR}/usr/include/ion.h ]
+if [ false && ! -e ${DESTDIR}/usr/include/ion.h ]
```

Comment:
**Question**: Is this intentional and do we want a comment explaining why?

Reply:
For the moment this project no longer has a build dependence on ION and it's a slow CI step, but there should be a ticket to re-integrate with ION BP transport after apl-fy24 merge.